src/seer/automation/codebase/repo_client.py

```diff
@@ -322,60 +322,91 @@ def _create_branch(self, branch_name):
 
         return ref
 
-    def _commit_file_change(self, change: FileChange, branch_ref: str):
-        contents = (
-            self.repo.get_contents(change.path, ref=branch_ref)
-            if change.change_type != "create"
-            else None
-        )
+    def _commit_file_change(
+        self, branch_ref: str, patch: FilePatch | None = None, change: FileChange | None = None
```

Comment:
```suggestion
        self, *, branch_ref: str, patch: FilePatch | None = None, change: FileChange | None = None
```
Given that we use this with kwargs only anyways and either field is required